network/types.go

```diff
@@ -73,6 +73,15 @@ type LinkRule struct {
 	Subnet net.IPNet
 }
 
+// RoutingPolicyType defines a certain routing policy to a network.
+type RoutingPolicyType string
+
+const (
+	// AllowAll whitelists all traffic through the data plane (except for the
+	// control traffic).
+	AllowAll = RoutingPolicyType("allow_all")
```

Comment:
In the other PR I see that we are checking for `AllowAll` policy and `default`. I think it'd be nice if we are explicit about the policy, and always have one, so maybe worth adding `DenyAll`. WDYT?

Reply:
Right, it makes sense. Should we make it a mandatory flag? We'd need to change all the tests. Right now, DenyAll would be the default.

Reply:
So I just added `DenyAll` and made a few updates to the docs. I will update the corresponding PR on testground repo and see how it looks best.